src/screens/OtherProfile/index.js

```diff
@@ -741,10 +749,12 @@ const styles = StyleSheet.create({
     marginLeft: 18
   },
   profileImage: {
-    width: 100,
-    height: 100,
+    width: 94,
+    height: 94,
     borderRadius: 100,
-    marginRight: 22
+    // marginLeft: 3,
+    marginTop: 3,
+    marginBottom: 3
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in the style of the profile image seem fine, but I noticed that the `marginLeft` property was commented out. If this property is not needed, consider removing it entirely to keep the code clean and avoid confusion.
```diff
-    // marginLeft: 3,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->